test/integration/README.md

```diff
@@ -0,0 +1,22 @@
+# Integration Testing
```

Comment:
Can you link to this readme from DEVELOPING.md? Also, the explanation of when the tests run etc., can go into the DEVELOPING.md.